src/Microsoft.AspNet.DataProtection/Internal/IActivator.cs

```diff
@@ -3,13 +3,13 @@
 
 using System;
 
-namespace Microsoft.AspNet.DataProtection
+namespace Microsoft.AspNet.DataProtection.Internal
```

Comment:
There's currently an issue with the netcore version in that it doesn't codegen strong named assemblies. Making these internals internal instead and putting them in a `.Internal` namespace.